scribble-doc/scribblings/scribble/config.scrbl

```diff
@@ -640,6 +640,20 @@ style:
 
  ]
 
+Additionally, the @filepath{racket.tex} Latex configuration
+includes several macros that you can redefine to adjust the
+output style of Racket code:
+
+@itemlist[
```

Comment:
Both these should say what they're called with (rendered arguments of `@racket` ?) so it's clearer how to override them. Reading about `SHyphen`, it seemed like a boolean parameter until I got to the example.